tests/test_grpc.py

```diff
@@ -93,6 +93,32 @@ def test_download_missing_file(mapdl, tmpdir):
         mapdl.download('__notafile__', target)
 
 
+def test_grpc_custom_ip():
+    from ansys.mapdl.core import launch_mapdl
+    
+    ip='127.0.0.2'
+    mapdl = launch_mapdl(ip=ip)
+
+    # Check the ip
+    # print(mapdl._ip)
+
+    # Check it also in the output file.
+    # print(mapdl.directory)
```

Comment:
```suggestion
    assert ip == mapdl.ip
```

Reply:
I do not trust mapdl.ip because it is set before running gRPC. I understand if everything is going well, it should reflect the real IP. But since it is a test, I have decided to go down the route of exctract the IP from the ouput file. I hope it does make sense.